extensions/panache/mongodb-panache-common/runtime/src/main/java/io/quarkus/mongodb/panache/PanacheUpdate.java

```diff
@@ -18,7 +18,7 @@
      * @param params params optional sequence of indexed parameters
      * @return the number of entities updated.
      */
-    public long where(String query, Object... params);
+    long where(String query, Object... params);
```

Comment:
I actually thought i had cleaned all those up and then saw I'd missed one.  i can roll that one back.  but generally, `public` shouldn't be added to interface methods because it's redundant.